__tests__/pages/provinces/index.test.tsx

```diff
@@ -57,6 +58,45 @@ describe("ProvincesPage", () => {
     ).toBeVisible();
   });
 
+  it("renders the provinces list kebutuhan entry count correctly", async () => {
+    const checkRenderKebutuhanEntryCount = async (
+      kebutuhan: string,
+      entryCount: number,
+    ) => {
+      await router.push(`/provinces?kebutuhan=${kebutuhan}`);
+      const { unmount } = render(
+        <ProvincesPage provincesList={provinceList} />,
+      );
+
+      const provinceLink = screen.getByRole("link", {
+        name: /informasi faskes & alkes untuk covid-19 di provinsi/i,
+      });
+
+      expect(
+        within(provinceLink).getByText(`${entryCount} Entri`),
+      ).toBeVisible();
+
+      unmount();
+    };
+
+    await checkRenderKebutuhanEntryCount(
+      "Ambulans",
+      provinceListItem.ambulansCount,
+    );
+    await checkRenderKebutuhanEntryCount(
+      "Rumah%20sakit",
+      provinceListItem.rsCount,
+    );
+    await checkRenderKebutuhanEntryCount(
+      "Oksigen",
+      provinceListItem.oksigenCount,
+    );
+    await checkRenderKebutuhanEntryCount(
+      "Donor%20plasma",
+      provinceListItem.donorPlasmaCount,
+    );
```

Comment:
Instead of calling a custom function within the test like this, you can simplify the test and make the failures more obvious to the developers by using [Jest's Each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) functionality. You can take a look at some other tests in this project that implements `.each()` as examples.

Please refactor this test to use `.each()` properly, thanks, @nkristoporus! 🙏